src/main/c/ssl.c

```diff
@@ -1626,6 +1626,49 @@ TCN_IMPLEMENT_CALL(jlong, SSL, getTime)(TCN_STDARGS, jlong ssl)
     return SSL_get_time(ssl_->session);
 }
 
+
+TCN_IMPLEMENT_CALL(jlong, SSL, getTimeout)(TCN_STDARGS, jlong ssl)
+{
+    SSL *ssl_ = J2P(ssl, SSL *);
+
+    if (ssl_ == NULL) {
+        tcn_ThrowException(e, "ssl is null");
+        return 0;
+    }
+
+    UNREFERENCED(o);
```

Comment:
the difference is that UNREFERENCED_STDARGS will unreferenced JNIEnv and jobject (which I think would be fine here as well). That said UNREFERENCED(o) is used in getTime(...) so maybe just leave it. WDYT ?